ddht/v5_1/alexandria/abc.py

```diff
@@ -87,9 +88,9 @@ async def send_get_content(
         node_id: NodeID,
         endpoint: Endpoint,
         *,
-        content_id: Hash32,
+        content_id: ContentID,
         start_chunk_index: int,
-        num_chunks: int,
+        max_chunks: int,
```

Comment:
`num` -> `max` is to account for padding?

Reply:
Changed to `max_` because it better matches the semantics of the way the content is provided.  When we request data we say "Give me data starting at chunk index `X` and up to `N` chunks after that".  The only firm requirement is that the responding proof contains the chunk at index `X`.  

So `max_chunks` seemed like a more appropriate name.... but I'm open to alternative suggestions since I recognize that `max_` also suggest that this will be validated which it isn't (and doesn't need to be).